src/main/java/core/basesyntax/dataio/CsvToStorage.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.dataio;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.operation.OperationMap;
+import core.basesyntax.operation.OperationStrategy;
+import core.basesyntax.operation.OperationStrategyImpl;
+import core.basesyntax.operation.Operations;
+import java.util.List;
+import java.util.Objects;
+
+public class CsvToStorage {
```

Comment:
let's refactor this service

create an interface FruitService with 2 methods:

void saveToStorage(List<String> data);

String getReportFromStorage();


create FruitServiceImpl class that implements FruitService and accepts `operationStrategy` as a constructor parameter (it should have `operationStrategy` field respectively)

create class Main with main() method and call your services one by one - there you will create OperationStrategyImpl and pass it into the new FruitServiceImpl